container/Dockerfile

```diff
@@ -0,0 +1,111 @@
+# BUILDER
+FROM alpine:latest AS builder
```

Comment:
While I understand the benefits of having a distro with a small footprint as a base, alpine is known to have its set of issues given it's based on musl and not glibc (I can see you already posted some workaround for dot). Recent [news about most prolific package manager leaving alpine](https://www.phoronix.com/news/Alpine-Linux-Prolific-Packager) on top of that make me wonder if we shouldn't be aiming at something else here (fedora/rhel ubi).

Reply:
I don't dislike the musl-busybox duo; in fact, it's the driving force behind my choice. Their lightweight nature is the main motivation for my decision.

Initially, my goal was to create a container that could be completely independent of any distribution. Since my application is written in Golang that produces by default static executables, ks-nav itself could operate without any additional dependencies. However, there are dependencies that are unavoidable.

Let's examine these dependencies:

* PostgreSQL is a significant one. For this ks-nav initial release, I prefer to stick with it due to its extensive testing and superior performance in both database construction and diagram creation.

* Currently, Radare2 is used for extracting symbols and cross-references. This task could potentially be achieved with other lightweight solutions, possibly in house made using golang. Over time, I might consider completely removing Radare2 from the loop.

* The introduction of the graphviz-dot dependency was due to a minor bug in the Golang implementation when building the `svg` format. Eventually, I decided to drop this dependency, as evidenced by the workaround implemented.

In summary, I am inclined to remain with Alpine for the time being. Eventually, I plan to transition to a distribution-free container once I've resolved all the issues preventing its immediate implementation.

Reply:
Ack, thanks for the thorough explanation on alpine vs others and postgres vs sqlite.